embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/spans/SpanService.kt

```diff
@@ -18,6 +19,7 @@ interface SpanService : Initializable {
      */
     fun createSpan(
         name: String,
+        autoTerminationMode: AutoTerminationMode,
```

Comment:
These should probably all default to NONE as well?

Reply:
I was in two minds about this but ended up thinking I'd rather have this mode required for any caller in production code so that we don't forget to set it, at the cost of making our test code more verbose. Do you think the risk of the other approach is worth it?